build/command.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+const { build } = require( "./tasks/build" );
+const yargs = require( "yargs/yargs" );
+const slimExclude = require( "./tasks/lib/slim-exclude" );
+
+const argv = yargs( process.argv.slice( 2 ) )
```

Comment:
Too bad we can't just use the native [`util.parseArgs` Node.js API](https://nodejs.org/api/util.html#utilparseargsconfig)...

Reply:
That would be nice, but we would have recreate the help command.